python/py-gmpy2/Portfile

```diff
@@ -5,7 +5,8 @@ PortGroup           python 1.0
 
 name                py-gmpy2
 epoch               1
-version             2.1.0
+version             2.1.2
+platforms           darwin
```

Comment:
this is the default value for platforms, so no reason to add this again

Reply:
Thank you for the review. I've made the requested changes.